index.bs

```diff
@@ -107,22 +105,18 @@ The following code queries the available local fonts, and logs the names and met
 
 ```js
 showLocalFontsButton.onclick = async function() {
-  // This sketch returns individual FontMetadata instances rather than families:
-  // In the future, query() could take filters e.g. family name, and/or options
-  // e.g. locale. A user agent can return all fonts, or show UI allowing selection
-  // of a subset of fonts.
   try {
     const array = await navigator.fonts.query();
 
     array.forEach(metadata => {
       console.log(metadata.postscriptName);
-      console.log(` full name: ${metadata.fullName}`);
-      console.log(` family: ${metadata.family}`);
-      console.log(` style: ${metadata.style}`);
+      console.log(\` full name: ${metadata.fullName}\`);
```

Comment:
@tabatkins Is it a Bikeshed bug that it needs this escaping?

Reply:
Yes, it is, and I plan to fix it this year, but it's unfortunately non-trivial. (The interaction of Markdown's "don't interpret anything inside of backticks" and HTML's "don't interpret anything inside of style/script" requires interweaving of the two parsers in a complex fashion that requires substantially rewriting *both*.)